### ISSUE TITLE
Hint for server url to add

### ISSUE THREAD

**Issue by schumaml** [JohnMHarrisJr/BoogDroid#6]
It isn't really obvious what I have to add in the URL field of a server - just https://bugzilla.example.com ? Or another of the well-known URLs of a Bugzilla instance, i. e. one of its scripts?

**Comment by JohnMHarrisJr** [comment]
I will add placeholder text to the field, you don't put the URL of one of the scripts, just the path to the base directory of Bugzilla.

**Comment by albfan** [comment]
Yes now you have xmlrpc and jsonrpc implemented which is \<baseur\l>+"xmlrpc.cgi" or \<baseurl\>+"jsonrpc.cgi". I will implement rest api based on work on https://github.com/JulienDev/BugDroid (I contact owner and he can give a hand, but he is aggree with reuse the code)

**Comment by albfan** [comment]
@schumaml what are your doubts exactly? We need to add a check that you do not add "jsonrpc.cgi" or "xmlrpc.cgi" or "rest/" at the end, but document it should be enough by now.

Adding a wiki would allow us to close this issue?

**Comment by schumaml** [comment]
The motivation for reporting the issue was mostly that nothing obvious happened when I tried this with https://bugzilla.gnome.org

The best way to handle this could be to be tolerant in regard to accepting URLs - meaning as long as it can get to a working API endpoint by traversing up from the URL provided, anything will work.

**Comment by albfan** [comment]
@schumaml already merged changes are precisely to integrate app with bugzilla.gnome.org. It uses the xmlrpc api that was not implemented. See now there's a check on server config to use json api. Leave unchecked to use xmlrpc (as needed for bugzilla.gnome.org)

In fact we are on gnome november bug squash https://wiki.gnome.org/Initiatives/BugSquashMonth so grab last version of app and give a hand if you want. 

https://github.com/albfan/BoogDroid/releases/tag/0.0.2b

Now you have search text filters on products, search text filters on bugs and you can add comments or change status (if you have permissions) See this is just a beta:

Normally we are on irc (#bugs channel), but you can use any of these links to ask for task or doubts 

https://wiki.gnome.org/AlbertoFanjul

Anyone is invited to join
